lessons/meta/local-run/index.md

```diff
@@ -0,0 +1,277 @@
+klic3:
+  vnorenyklic: Mapa může obsahovat další mapu i takto.
+```
+
+### Povinné informace
+
+Teď když už se vyznáme ve formátu YAML, můžeme začít vytvářet soubor ``info.yml``.
+Celá definice kurzu je jedna velká mapa, kterou si postupně popíšeme a vyplníme.
+Pro potřeby kurzu je [zde]({{static("info.yml")}}) připravená šablona, kterou můžeš použít.
+
+Nejdřív prvních pár základních povinných údajů:
+
+* ``title`` slouží pro název kurz (nepovinně lze dodefinovat pomocí ``subtitle``)
+* ``description`` slouží pro krátký popis kurzu (který se zobrazí v seznamu kurzů)
+* ``long_description`` slouží pro dlouhý popis kurzu, který se zobrazí na stránce kurzu
+
+A teď už jen nepovinné údaje:
+
+* ``place`` slouží pro označení místa
+* ``time`` slouží pro informaci o času (není nutný žádný specifický formát)
+
+Jestli chceš, aby pro kurz šel vygenerovat iCal soubor s plánem lekcí, musíš dále poté vyplnit údaj ``default_time``.
+Tato hodnota musí být mapa, která obsahuje dva klíče ``start`` a ``end``, kde bude čas lekcí.
+Čas musí být ve formátu ``HH:MM`` a musí být obalen uvozovkami nebo apostrofy, takže třeba takhle:
+
+```yaml
+default_time:
+  start: '18:00'
+  end: '20:00'
+```
+
+Poslední nepovinná hodnota než se dostaneme k obsahu kurzu jsou proměné, které se definují klíčem ``vars`` a musí být také mapa.
+Proměné mohou upravovat obsah lekcí a stránek kurzu a pokud budeš vytvářet nebo upravovat materiály, můžeš si i definovat vlastní.
+
+První proměnou, kterou můžeš použít je ``coach-present``.
+Pokud je tvůj kurz s lektorem nebo koučem, nastav tuto proměnou na hodnotu ``true``.
+
+Druhou proměnou, kterou můžeš použít je ``user-gender``.
+Pokud víš, že na tvém kurzu budou lidé jen jednoho pohlaví, můžeš nastavit materiály (které tak byly napsány), aby vykreslily správně formátovaný obsah.
+Nastavíš to pomocí písmenka ``f`` pro ženy, ``m`` pro muže.
+Pokud proměnou nevyplníš, materiály vykreslí obě varianty.
+
+Další proměné, které můžeš použít jsou ``pyladies`` nebo ``mi-pyt``.
+Ty využij (nastavením na hodnotu ``true``), pokud odvozuješ svůj kurz od těchto základních kurzů – viz níže.
+
+### Plán na základě kanonických materiálů
+
+Na vytvoření obsahu máš dvě základní možnosti.
+Buď využiješ plán existujícího kanonického kurzu nebo si nadefinuješ svůj vlastní.
+Zde je popsán postup odvozování.
+
+Nejdřív si vyber kanonický kurz, ze kterého chceš svůj kurz odvodit.
+Kanonické kurzy najdeš ve složce ``courses``.
+Vybraný kanonický kurz nastav v souboru ``info.yml`` klíčem ``derives`` – napiš do něj název složky kurzu (např. ``pyladies`` nebo ``mi-pyt``).
+Dále pak rozšiř seznam proměných s proměnou, která se bude jmenovat stejně jako kurz a hodnotou ``true``.
+
+A teď už zbývá poslední věc a to je sestavení programu kurzu.
+Kurz se skládá z jednotlivých lekcí a každá lekce se skládá z jednotlivých materiálů.
+Lekce nadefinuješ klíčem ``plan`` v souboru ``info.yml``.
+Ten musí být seznam dalších slovníků.
```

Comment:
map? možná bych se tomu vyhnul a všude používal seznam a slovník

Reply:
Jop, ani nevím proč jsem tam původně napsal mapu, sjednotím to všude na slovníky a seznamy